arnold/api/api_v1/endpoints/case.py

```diff
@@ -13,7 +13,6 @@
     "/case/",
     response_description="Create a new case document",
     status_code=status.HTTP_201_CREATED,
-    response_model=Case,
```

Comment:
I think the swagger uses these annotations? Might be nice with a JSONResponse type hint? Not a pro at this though.